.gitignore

```diff
@@ -1,3 +1,6 @@
 .DS_Store
 node_modules/
 temp/
+
```

Comment:
A global ignore (on some binary files) has bitten me before, when trying to figure out why things worked at one computer but not another.
The fact that there is no such thing as a global ``.gitignore`` out of the box (you have to add it to a global ``.gitconfig``) also makes me somewhat hesitant.